src/edn.rs

```diff
@@ -0,0 +1,42 @@
+pub fn to_edn_keyword(field_name: String) -> String {
+    let mut keyword = field_name
+        .to_lowercase()
+        .replace("___", "/")
```

Comment:
Adding here the reference I said in the other PR, `maybe_replace` had a better performance than replace in the other crate https://github.com/naomijub/edn-rs/blob/master/src/deserialize/parse.rs#L205
How ever here it may be slower because it is almost sure that we have `___`, `__`, `_`

Reply:
Issue: https://github.com/otaviopace/edn-derive/issues/11

Reply:
But i am guessing it wont improve much because structs use this replaces a lot